Gopkg.toml

```diff
@@ -68,6 +68,10 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/e2etests"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/exporterkit"
+
```

Comment:
Note that the branch still has to be updated with a vendor update. Vendoring is currently broken but is expected to be fine again soon.

Reply:
This is updated now.